src/app/game-state/character.ts

```diff
@@ -114,83 +126,95 @@ export class Character {
       value: 1,
       lifeStartValue: 1,
       aptitude: 1,
+      aptitudeMult: 1,
```

Comment:
Legacy save files won't have these, so we may need something in the setProperties to make sure they exist before they get used.

Reply:
Right, I'll probably turn it into a method since it's really just a derived stat.